docs/content/docs/development/guide.md

```diff
@@ -108,6 +108,14 @@ So if the CRD is changed, you have to delete the CRD resource manually, and re-i
 kubectl delete crd flinkdeployments.flink.apache.org
 ```
 
+### Mounts
+
+The operator supports to specify the volume mounts. The default mounts to hostPath can be activated by the following command. You can change the default mounts in the `helm/flink-operator/values.yaml`
+
+```bash
+helm install flink-operator helm/flink-operator --set operatorVolumeMounts.create=true --set operatorVolumes.create=true 
+```
+
```

Comment:
Maybe later we should move this to the Helm page instead, but this is good for for now

Reply:
Agree